department-of-veterans-affairs/va.gov-cms
# VACMS-15882: Adds default pager heading

## Description

Updates [configurable pager heading](https://www.drupal.org/project/drupal/issues/3333401) patch to fallback to a default heading (h3) for pagers rendered without a view and other edge cases.

#10111 has the back story
Closes: #15201, #15099

## Testing done
Performed manual testing in Q/A and confirmed pager render issues no longer occur on affected pages.

## Screenshots
[PostAPI Queue Page](https://pr15948-525mxyhvwiwllgyrktivo6j4ftn4fis4.ci.cms.va.gov/admin/config/post-api/queue)
![VACMS-15882-20231101-1156-post-api-queue-pager](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/607178/b189d4ed-cb66-4ea7-bbe6-ef3dee71e80c)

[Health Care Service Taxonomy Overview](https://pr15948-525mxyhvwiwllgyrktivo6j4ftn4fis4.ci.cms.va.gov/admin/structure/taxonomy/manage/health_care_service_taxonomy/overview)
![VACMS-15882-20231101-1158-taxonomy-view-pager](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/607178/50616f14-bd2c-4cfb-9c37-31e6c6f1e4d3)


## QA steps

What needs to be checked to prove this works?

- Navigate to https://pr15948-525mxyhvwiwllgyrktivo6j4ftn4fis4.ci.cms.va.gov/admin/structure/taxonomy/manage/health_care_service_taxonomy/overview
  - Scroll to bottom of page
  - Verify no instance of `< id="pagination-heading" class="visually-hidden">Pagination` exists
- Navigate to  https://pr15948-525mxyhvwiwllgyrktivo6j4ftn4fis4.ci.cms.va.gov/admin/config/post-api/queue
  - Scroll to bottom of page
  - Verify no instance of `< id="pagination-heading" class="visually-hidden">Pagination` exists


### Select Team for PR review
